sdks/python/scripts/run_expansion_services.sh

```diff
@@ -133,4 +136,7 @@ case $STARTSTOP in
     fi
     ;;
 esac
-flock -u 200
+
+if [[ $CHECK_FLOCK -eq 0 ]]; then
```

Comment:
Can you add some comments to clarify why this and other conditions introduced here are needed ?

Reply:
Done.